src/encoded/types/user.py

```diff
@@ -17,7 +46,7 @@
 )
 class User(SMAHTItem, SnovaultUser):
     """ Overridden user class, adding the Submission Center and Consortium attribution """
-    item_type = 'smaht-user'
+    item_type = 'user'
```

Comment:
Here you renamed both the class and the item type, unlike elsewhere that you did not.